script/RumpelConfig.sol

```diff
@@ -201,6 +213,42 @@ library RumpelConfig {
 
         return configs;
     }
+
+    // Mellow Re7 ----
+    function getMellowRe7GuardProtocolConfigs() internal pure returns (ProtocolGuardConfig[] memory) {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](2);
+
+        // Mellow LRT Vault re7LRT
+        configs[0] = ProtocolGuardConfig({target: MAINNET_RE7LRT, allowedSelectors: new bytes4[](2)});
+        configs[0].allowedSelectors[0] = IMellow.deposit.selector;
+        configs[0].allowedSelectors[1] = IMellow.registerWithdrawal.selector;
+
+        // Mellow WBTC Vault re7RWBTC
+        configs[1] = ProtocolGuardConfig({target: MAINNET_RE7RWBTC, allowedSelectors: new bytes4[](2)});
+        configs[1].allowedSelectors[0] = IMellow.deposit.selector;
+        configs[1].allowedSelectors[1] = IMellow.registerWithdrawal.selector;
+
+        return configs;
+    }
+
+    function getMellowRe7GuardTokenConfigs() internal pure returns (TokenGuardConfig[] memory) {
+        TokenGuardConfig[] memory configs = new TokenGuardConfig[](3);
+
+        configs[0] = TokenGuardConfig({token: MAINNET_WBTC, allowTransfer: true, allowApprove: true});
+        configs[1] = TokenGuardConfig({token: MAINNET_RE7LRT, allowTransfer: true, allowApprove: false});
+        configs[2] = TokenGuardConfig({token: MAINNET_RE7RWBTC, allowTransfer: true, allowApprove: false});
```

Comment:
Do we need to allow approve here so that you can repay your vault position. I'd guess you need to approve for the vault to transfer your shares from you back to the vault. Would apply to both.

Reply:
We didn't allow approve for the other mellow vaults (e.g. `MAINNET_RSUSDE`). My understanding is that, since the token itself is the vault, it always has auth to just directly burn your tokens. Worth double checking, though

Reply:
reviewed the contracts and you are right. the registerWithdrawal uses the internal _transfer of the erc20 to bypass needing approval.